apps/ios/src/Podfile

```diff
@@ -27,6 +27,10 @@ fi
 use_flipper!(false)
 use_test_app! do |target|
   target.app do
+
+    pod 'FURNShimmer', :path => '../../../packages/components/Shimmer/FURNShimmer.podspec'
+
```

Comment:
@tido64 In the end, this was required, as the native files weren't imported without manually adding this

Reply:
Does adding `"@fluentui-react-native/shimmer": "*"` to `/apps/ios/package.json` help? I suspect what's happening is that rn-cli is unable to find the `.podspec` because the package isn't installed anywhere. If you look at `Podfile.lock`, all of the paths are under `node_modules`.

Reply:
Changing just shimmer to * in the package.json for `apps/ios/` and `apps/fluent-tester` did not work.

To answer your other question, I don't think shimmer is published to the npm registry yet. I didn't do any work to explicitly do that. I'll have to check for the other components.